debug_gym/gym/envs/env.py

```diff
@@ -257,8 +257,24 @@ def _reset_env_state(self):
 
     def set_entrypoints(self, entrypoint: str, debug_entrypoint: str | None = None):
         if entrypoint:
-            self.entrypoint = entrypoint
-            debug_entrypoint = debug_entrypoint
+            is_python = "python " in entrypoint
+
+            self.entrypoint = self._prepare_entrypoint(entrypoint)
+
+            if debug_entrypoint is None:
+                debug_entrypoint = (
+                    entrypoint.replace("python ", "python -m pdb ", 1)
+                    if is_python else entrypoint
```

Comment:
Minor: `is_python` is true only if `entrypoint` has "python " and `.replace` is going to replace only the instances of "python " anyway.